tests/test_more.py

```diff
@@ -5927,16 +5928,16 @@ def test_basic(self):
         # Example calculation from:
         # https://en.wikipedia.org/wiki/Discrete_Fourier_transform#Example
         xarr = [1, 2 - 1j, -1j, -1 + 2j]
-        Xarr = [2, -2 - 2j, -2j, 4 + 4j]
-        self.assertTrue(all(map(cmath.isclose, mi.dft(xarr), Xarr)))
-        self.assertTrue(all(map(cmath.isclose, mi.idft(Xarr), xarr)))
+        xarr2 = [2, -2 - 2j, -2j, 4 + 4j]
```

Comment:
Or maybe `yarr` or something else, please feel free to propose better variants.

Reply:
No.